docs/msi-mpg-series-coreliquid-guide.md

```diff
@@ -0,0 +1,3 @@
+# MSI MPG Series CORELIQUID
+
+This driver only supports reporting status. Only the K360 model is supported.
```

Comment:
This file is important and needs to follow the same general template of the other `*guide.md` files.